core/reducers/blueprints.js

```diff
@@ -250,10 +221,34 @@ const blueprints = (state = [], action) => {
               return Object.assign({}, blueprint, {
                 present: Object.assign({}, blueprint.past.shift(), {
                   localPendingChanges: [],
-                  workspacePendingChanges: { addedChanges: [], deletedChanges: [] }
+                  workspacePendingChanges: []
```

Comment:
The same change needs to be applied to all `webspacePendingChanges`. Otherwise this prop will get old value and `this.props.blueprint.workspacePendingChanges.map` in `PendingChanges.js` will throw an error and pending changes cannot get rendered in Pending Changes page.
![workspacependingchanges](https://user-images.githubusercontent.com/672230/51235018-64a1bf00-19a9-11e9-9a78-611e4f188e6f.png)
```shell
diff --git i/core/reducers/blueprints.js w/core/reducers/blueprints.js
index 5a4521d4..698ace86 100644
--- i/core/reducers/blueprints.js
+++ w/core/reducers/blueprints.js
@@ -27,7 +27,7 @@ const blueprints = (state = [], action) => {
             past: [],
             present: Object.assign({}, action.payload.blueprint, {
               localPendingChanges: [],
-              workspacePendingChanges: { addedChanges: [], deletedChanges: [] }
+              workspacePendingChanges: []
             }),
             future: []
           }
@@ -50,7 +50,7 @@ const blueprints = (state = [], action) => {
                 past: [],
                 present: Object.assign({}, action.payload.blueprint, {
                   localPendingChanges: [],
-                  workspacePendingChanges: { addedChanges: [], deletedChanges: [] }
+                  workspacePendingChanges: []
                 }),
                 future: []
               }
@@ -131,7 +131,7 @@ const blueprints = (state = [], action) => {
                 past: [],
                 present: Object.assign({}, action.payload.blueprint, {
                   localPendingChanges: [],
-                  workspacePendingChanges: { addedChanges: [], deletedChanges: [] }
+                  workspacePendingChanges: []
                 }),
                 future: []
               });
```

Reply:
Thanks for catching that!